src/__tests__/wrap.v3.e2e.test.ts

```diff
@@ -468,7 +468,7 @@ describe("wrap", () => {
           unwrap: false,
           batched: true,
         });
-        // console.log(fs.readFileSync(outputFile.name, { encoding: "utf8" }));
+        console.log(fs.readFileSync(outputFile.name, { encoding: "utf8" }));
```

Comment:
Is it printing in test? this is quite irritating.. Can remove this altogether?

Reply:
Done.